src/components/content/Content.tsx

```diff
@@ -0,0 +1,53 @@
+import { Footer, Header, loadTranslations, Typography, useTheme, useTranslation } from '@okp4/ui'
+import type { ThemeContextType, UseTranslationResponse } from '@okp4/ui'
+import lightCosmos from '@okp4/ui/lib/assets/images/cosmos-clear.png'
+import darkCosmos from '@okp4/ui/lib/assets/images/cosmos-dark.png'
+import { translationsToLoad } from '../../i18n/index'
+import { Faucet } from '../faucet/Faucet'
+
+const languages = [
+  {
+    name: 'English',
+    lng: 'en'
+  },
+  {
+    name: 'Français',
+    lng: 'fr'
+  }
+]
+
+const Okp4Link = (): JSX.Element => {
+  const { t }: UseTranslationResponse = useTranslation()
+
+  return (
+    <Typography as="p" color="highlighted-text" fontSize="x-small" fontWeight="xlight" noWrap>
```

Comment:
It might be more appropriate to use a `div` here because it is not really a paragraph element since it is not wrapped in a parent tag and its direct child is a `div`.
Maybe we could use the first `Typography` element as a `div` and the second as a `span` for example

Reply:
@fredericvilcot Here the `Okp4Link` is embedded in the `Footer` s div, so for me it was OK to keep it as a paragraph since the 2nd `Typography` element has the default `as` `span` prop, but I'm opened for changes if you prefer !